src/components/Features.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * Featurescomponent
+ * This is the features and some information
+ */
+
+/* Imports -------------------------------------------------------------------*/
+
+import Inferno from 'inferno';
+import Component from 'inferno-component';
+
+/* Local variables -----------------------------------------------------------*/
+
+const version = '^3.0.0';
+
+/* Component -----------------------------------------------------------------*/
+
+class VersionComponent extends Component {
```

Comment:
You need to put a meaningful name for this component, this is still the template's name.